geoid/__init__.py

```diff
@@ -122,7 +124,7 @@
     'sdsec': 5,  # State-School District (Secondary)
     'sduni': 5,  # State-School District (Unified)
     'sldl': 3,  # State Legislative District Lower
-    'sldu': "3",  # State Legislative District Upper
```

Comment:
The difference between integers and strings is meaningful here -- it means that the field can occasionally have a string value, so the template for generating geoids must have {:s} instead of {:d}